OrbitClientModel/CaptureSerializer.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientModel/CaptureSerializer.h"
+
+#include <google/protobuf/io/coded_stream.h>
```

Comment:
Do you mean changing the order of the includes? clang format order the headers like this when I save the file so I can't really move them around

Reply:
Ah, okay. 
I'd use use "..." instead of <...> for everything except standard library header files.  If clang_format then still has an opinion don't fight it ...

Reply:
Ok, I will update them to use "..." instead

Reply:
Ah! And with this change _clang_format_ moves it below! great, thanks!

Reply:
Done!